gossip/src/crds_value.rs

```diff
@@ -1,13 +1,15 @@
 use {
     crate::{
-        cluster_info::{MAX_ACCOUNTS_HASHES, MAX_CRDS_OBJECT_SIZE},
+        cluster_info::MAX_ACCOUNTS_HASHES,
         contact_info::ContactInfo,
         deprecated,
         duplicate_shred::{DuplicateShred, DuplicateShredIndex, MAX_DUPLICATE_SHREDS},
-        epoch_slots::{CompressedSlots, EpochSlots, MAX_SLOTS_PER_ENTRY},
+        epoch_slots::{EpochSlots, MAX_SLOTS_PER_ENTRY},
         legacy_contact_info::LegacyContactInfo,
     },
     bincode::{serialize, serialized_size},
+    bv::BitVec,
+    itertools::{Itertools, MinMaxResult::*},
```

Comment:
wildcard imports are discouraged:
https://rust-lang.github.io/rust-clippy/master/index.html#/wildcard_imports

Can you please use the fully qualified enum variants? e.g. `MinMaxResult::OneElement`, etc.

Reply:
Done.